samples/matter/light_bulb/Kconfig.sysbuild

```diff
@@ -13,7 +13,7 @@ config NETCORE_IPC_RADIO_IEEE802154
 config NETCORE_IPC_RADIO_BT_HCI_IPC
 	default y if (BOARD_NRF7002DK_NRF5340_CPUAPP || BOARD_NRF5340DK_NRF5340_CPUAPP)
 
-config MATTER_FACTORY_DATA
```

Comment:
@ArekBalysNordic 
should it be also updated in applications?
applications/matter_weather_station,
applications/matter_bridge

Reply:
I've already done it in my PR so it will be added anyway. I need to wait to merge this PR and then I can align and test all samples and applications.